assets/js/theme/search.js

```diff
@@ -221,7 +221,11 @@ export default class Search extends CatalogPage {
 
         q$('body').insertAdjacentElement('beforeend', $searchResultsMessage)
 
-        setTimeout(() => $searchResultsMessage.focus(), 100)
+        setTimeout(() => {
+            $searchResultsMessage.focus()
+            q$('html').scrollTop = 0
+            q$('body').scrollTop = 0
```

Comment:
Why do you need to scroll both `html` and `body` ?  If you need to execute some code with a slightly delay, try use `requestAnimationFrame` first instead of using setTimeout with magic delay value - I know that you just added `scrollTop` tho

Also, I encourage to use vanilla js code instead of jquery :)

Reply:
I love vanilla JS better too, just wanted to keep it consistent. But i'll change it no worries